tests/build/build_templates.go

```diff
@@ -602,3 +615,41 @@ func getImageWithDigest(c *framework.ControllerHub, componentName, applicationNa
 	}
 	return fmt.Sprintf("%s@%s", url, digest), nil
 }
+
+func updateBuildPipelineSelector(kubeadminClient *framework.ControllerHub, testNamespace string) error {
```

Comment:
A few thoughts on this function:

* make this function name be more specific
* make it more generic for updating pipeline bundle in `BuildPipelineSelectors`. That is, rename it to `updatePipelineBundleInBuildPipelineSelector` and accept two additional arguments `bundle string` and `pipelineName string`

Reply:
this function is temporary till we have build-pipeline-selector removal is completed and soon we will be moving to `build-config` instead of `build-pipeline-selector` in tests, so I think it would not be wise to invest in this function definition making it more generic.